components/application-gateway/internal/authorization/oauth/oauthclient_test.go

```diff
@@ -238,3 +216,19 @@ func TestOauthClient_InvalidateAndRetry(t *testing.T) {
 		tokenCache.AssertExpectations(t)
 	})
 }
+
+func checkAccessTokenRequest(t *testing.T, r *http.Request) {
+	r.ParseForm()
+
+	assert.Equal(t, "test", r.PostForm.Get("client_id"))
+	assert.Equal(t, "test", r.PostForm.Get("client_secret"))
```

Comment:
I would suggest providing different values for id and secret, for example `testID` and `testSecret`, this will improve readability in further assertions.